README.md

```diff
@@ -29,17 +28,21 @@ Hartnett](mailto:edward.hartnett@noaa.gov)
 ## Prerequisites
 
 The following libraries are required:
-- libpng
-- libjasper
+
+This package requires the following third party libraries:
+- [Jasper](http://www.ece.uvic.ca/~mdadams/jasper/)
+- [libpng](http://www.libpng.org/pub/png/libpng.html)
+- [libz](http://www.gzip.org/zlib/)
```

Comment:
I think this link should be https://www.zlib.net/ (current one doesn't work)

Reply:
and I would rename libz->zlib

Reply:
Done.